csi-plug-n-play/sidecars/soda-proxy/Readme.md

```diff
@@ -0,0 +1,37 @@
+## soda-proxy
+
+soda-proxy is a simple http server which helps to call the api's of SodaFoundation in a simpler way, it bypasses the authentication and directly helps to connect the osdsapi-server using the client provided by sodafoundation/api.
+
+### Build
+Please follow the below steps to build and run the proxy
+
+```go
+go get github.com/sodafoundation/nbp
+
+git checkout csipnp_dev 
+
+cd $GOPATH/src/github.com/sodafoundation/nbp/csi-plug-n-play/sidecars/soda-proxy
+
+go build cmd/proxy.go
+```
+
+Before running soda-proxy you need to export the below variables.
+
+```go
+
+export OPENSDS_ENDPOINT=http://{your_host_ip}:50040
+export OPENSDS_AUTH_STRATEGY=keystone
+export OS_AUTH_URL=http://{your_host_ip}/identity
+export OS_USERNAME=admin
+export OS_PASSWORD=opensds@123
```

Comment:
Can we avoid this plain text or mention the key env var. Also these env vars can be taken from any config or..?

Reply:
these are same env variable which are required for setting up osdsctl https://github.com/sodafoundation/api/wiki/SODA-Projects-Cluster-Installation-through-Ansible#how-to-test-soda-projects-cluster

Reply:
the same info has been updated in the Readme